docs/publication.md

```diff
@@ -0,0 +1,48 @@
+# How to publish issues on Phabricator
+
+## Build analysis output
+
+The code-review bot currently supports 6 different formats to report issues (clang-tidy, clang-format, zero coverage, coverity, infer and mozlint).
+We are in the process of standardizing toward [a single format described here](analysis_format.md).
+
+New analyzers are strongly encouraged to use that format, as it's supported directly in the bot. Other formats, or more detailed versions will need an update in the bot's code itself.
```

Comment:
Since our end goal is to have a standard format only, why don't we avoid mentioning the alternative formats altogether? This way we avoid tempting people to use them, and we reduce the cognitive overload for them (the users of the project don't really need to know these details, they just want to know how to create a new analyzer).